synapse/storage/schema/__init__.py

```diff
@@ -14,7 +14,7 @@
 
 # When updating these values, please leave a short summary of the changes below.
 
-SCHEMA_VERSION = 63
+SCHEMA_VERSION = 64
```

Comment:
@MadLittleMods @erikjohnston please read the comment on line 15 :angry:

Reply:
🙇 Updated in https://github.com/matrix-org/synapse/pull/10917